doc/tracepath.xml

```diff
@@ -162,28 +162,28 @@ root@mops:~ # tracepath -6 3ffe:2400:0:109::2
  2:  3ffe:2400:0:109::2               463.514ms reached
      Resume: pmtu 1480 hops 2 back 2
     </literallayout>
-    <para>The first column shows TTL of the probe, followed by
-    colon. Usually value of TTL is obtained from reply from
-    network, but sometimes reply does not contain necessary
+    <para>The first column shows the TTL of the probe, followed by
+    colon. Usually the value of TTL is obtained from the reply from
+    the network. But sometimes it does not contain the necessary
```

Comment:
I'd keep here the original version: `from network, but sometimes` (compound sentence isn't forbidden in English :)).

Reply:
Agreed. I reverted this part.